building.md

```diff
@@ -1,30 +1,22 @@
 ## Building the plugin
 
-We use gradle to build the plugin, and require version 2.1 or greater (`gradle -version`).
+We use ant to build the plugin; to check your ant install, run `ant -version`.
```

Comment:
The latest available is `1.9.7`; the current version on travis is `1.8.2`. That seems to meet our needs - and we're not using anything 1.9 specific - so we can take that as our current min.